.travis.yml

```diff
@@ -62,9 +62,6 @@ matrix:
     - arch: arm64
       env:
         - JOB="3.7, arm64" PYTEST_WORKERS=8 ENV_FILE="ci/deps/travis-37-arm64.yaml" PATTERN="(not slow and not network and not clipboard)"
-    - dist: bionic
```

Comment:
How come we have removed this @VirosaLi did we move 3.9 build to Azure?

Reply:
> How come we have removed this @VirosaLi did we move 3.9 build to Azure?

It was removed from allow_failures.

Reply:
Ahh thanks misread the diff